db/seeds.rb

```diff
@@ -1,4 +1,12 @@
-# Create a few default tags.
-Tag::DEFAULT_TAGS.each do |name|
-  Tag.create!(name: name)
+unless defined?(Dradis::Pro)
+  # Create a few default tags.
+  Tag::DEFAULT_TAGS.each do |name|
+    Tag.find_or_create_by(name: name)
```

Comment:
In the CE case I don't think we need to change this, since it'll only be run once and when seeding, the tags are very likely to not exist yet